tests/e2e/multi-apps/chainsaw-test.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: multi-apps
+spec:
+  description: This e2e test runs a multi-apps scenario
+  skipDelete: true
+  steps:
+  - name: Prepare destination
+    try:
+    - script:
+        timeout: 60s
+        content: |
+          helm repo add grafana https://grafana.github.io/helm-charts
+          helm repo update
+          helm install e2e-tests grafana/tempo -n traces --create-namespace --set tempo.storage.trace.block.version=vParquet4 --version 1.10.1
+    - assert:
+        file: assert-tempo-running.yaml
+  - name: Wait for destination to be ready
+    try:
+      - script:
+          timeout: 60s
+          content: ../../common/wait_for_dest.sh
+  - name: Install Odigos
+    try:
+    - script:
+        content: ../../../cli/odigos install --version e2e-test
+        timeout: 60s
+    - assert:
+        file: assert-odigos-installed.yaml
+  - name: Install Demo App
+    try:
+    - script:
+        timeout: 100s
+        content: |
+          docker pull keyval/odigos-demo-inventory:v0.1
+          docker pull keyval/odigos-demo-membership:v0.1
+          docker pull keyval/odigos-demo-coupon:v0.1
+          docker pull keyval/odigos-demo-inventory:v0.1
+          docker pull keyval/odigos-demo-frontend:v0.2
+          kind load docker-image keyval/odigos-demo-inventory:v0.1
+          kind load docker-image keyval/odigos-demo-membership:v0.1
+          kind load docker-image keyval/odigos-demo-coupon:v0.1
+          kind load docker-image keyval/odigos-demo-inventory:v0.1
+          kind load docker-image keyval/odigos-demo-frontend:v0.2
```

Comment:
the image tag is not consistent (0.1 vs 0.2).
I think we should align with https://github.com/odigos-io/simple-demo/blob/main/kubernetes/deployment.yaml which is using v0.1.11. Can we simply `kubectl apply` the simple demo deployment which we use for the getting started?

Reply:
@blumamir You can look in the other chainsaw file - I raised the same thing and Eden answered there.